src/python/pants/reporting/html_reporter.py

```diff
@@ -430,7 +430,13 @@ def _render_tool_detail(self, workunit, title, class_prefix='greyed', initially_
 
   def _emit(self, s):
     """Append content to the main report file."""
-    if os.path.exists(self._html_dir):  # Make sure we're not immediately after a clean-all.
+    report_file_probably_writable = (
```

Comment:
we can remove this too, as we are removing any `RunTracker` changes from this.

Reply:
Reverted!